templates/vars/vnet/ithc.json

```diff
@@ -51,6 +51,31 @@
           "service": "Microsoft.Storage"
         }
       ]
-    }
+    },
+    "routetableName": {
+      "value": "aks-ithc-rt"
```

Comment:
better to fix this than hack it

Reply:
Cheers - now updated & deployed with aks-route-table